### PR TITLE
Overdue orders report back end

### PR DESCRIPTION
**Pull request by atcampbell**

### CODE REVIEW

File: src/Domain.LinnApps/ViewModels/OverdueOrderLine.cs [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+        public int NoStockQuantity { get; set; }
+
+        public int? AllocVal { get; set; }
+
+        public int? RnSDVal { get; set; }
+
+        public int? RnSNdVal { get; set; }
+
+        public int? RnNsVal { get; set; }
```
Comment (andrewfraser73) [inline src/Domain.LinnApps/ViewModels/OverdueOrderLine.cs]: You could rename some of these columns to be  verbose on the acronymns we know like "RequiredNowNoStockValue"

Comment (andrewfraser73) [reply src/Domain.LinnApps/ViewModels/OverdueOrderLine.cs]: Rtm = RequiredThisMonth, Rnm = RequiredNextMonth

File: src/Service/Modules/Reports/OrdersReportsModule.cs [hunk 2]
```diff
@@ -32,5 +34,27 @@ private object ManufacturingCommitDateReport()
...
+        private object OverdueOrdersReportOptions()
+        {
+            return this.Negotiate.WithModel(ApplicationSettings.Get()).WithView("Index");
+        }
+
+        private object OverdueOrdersReport()
+        {
+            var resource = this.Bind<OverdueOrdersReportRequestResource>();
+            var results = this.reportService.GetOverdueOrdersReport(
+                resource.JobId,
```
Comment (andrewfraser73) [inline src/Service/Modules/Reports/OrdersReportsModule.cs]: Do you have to specify the Job Id in the overdue orders request or can you just get it from the Accounting Company?  I don't specify the job in production back orders but just get the latest one from that accounting company

File: src/Domain.LinnApps/ViewModels/OverdueOrderLine.cs (hunk 1 above)
Comment (atcampbell) [reply src/Domain.LinnApps/ViewModels/OverdueOrderLine.cs]: What would the Nd and a single D stand for?

File: src/Service/Modules/Reports/OrdersReportsModule.cs (hunk 2 above)
Comment (atcampbell) [reply src/Service/Modules/Reports/OrdersReportsModule.cs]: Ahh yeah I'll have a look at that

File: src/Domain.LinnApps/ViewModels/OverdueOrderLine.cs (hunk 1 above)
Comment (atcampbell) [reply src/Domain.LinnApps/ViewModels/OverdueOrderLine.cs]: like RnSDVal and RnSNdVal

Comment (andrewfraser73) [reply src/Domain.LinnApps/ViewModels/OverdueOrderLine.cs]: ND = Not Despatchable
D = Dispatchable 

So RnSDVal is Required Now Stock Despatachable Value i.e. the Can Go value from sales stats

RnSNdVal is Required Now Stock Not Despatchable Value i.e. is the Could go value and there is another column with reasons which can be multiple including Supply In Full, Credit etc

does this help?

Comment (atcampbell) [reply src/Domain.LinnApps/ViewModels/OverdueOrderLine.cs]: Thanks!

**Pull request merged**